rmgpy/statmech/conformerTest.py

```diff
@@ -242,15 +250,15 @@ def test_getMomentOfInertiaTensor(self):
         Test the Conformer.getMomentOfInertiaTensor() method.
         """
         I = self.conformer.getMomentOfInertiaTensor()
-        self.assertAlmostEqual(I[0,0]*constants.Na*1e23, 20.65968, 4)
-        self.assertAlmostEqual(I[0,1]*constants.Na*1e23, -7.48115, 4)
-        self.assertAlmostEqual(I[0,2]*constants.Na*1e23, -3.46416, 4)
-        self.assertAlmostEqual(I[1,0]*constants.Na*1e23, -7.48115, 4)
-        self.assertAlmostEqual(I[1,1]*constants.Na*1e23, 13.53472, 4)
-        self.assertAlmostEqual(I[1,2]*constants.Na*1e23, -5.48630, 4)
-        self.assertAlmostEqual(I[2,0]*constants.Na*1e23, -3.46416, 4)
-        self.assertAlmostEqual(I[2,1]*constants.Na*1e23, -5.48630, 4)
-        self.assertAlmostEqual(I[2,2]*constants.Na*1e23, 22.84296, 4)
+        self.assertAlmostEqual(I[0, 0]*constants.Na*1e23, 20.65968, 4)
```

Comment:
additional spaces around the `*` operator

Reply:
Modified